telluric/plotting.py

```diff
@@ -146,6 +148,14 @@ def plot(feature, mp=None, style_function=None, **map_kwargs):
 
 class NotebookPlottingMixin:
     def _repr_html_(self):
+        # These imports are here to avoid cyclic references
+        from telluric.collections import BaseCollection
+        from telluric.features import GeoFeatureWithRaster
+        if isinstance(self, BaseCollection):
+            if isinstance(self[0], GeoFeatureWithRaster):
```

Comment:
maybe we should extract it to a method, 

`isFeatureCollectionWithRasters`

so we could use this logic in other places as well

Reply:
I agree but not as part of this PR